azurerm/internal/services/loganalytics/log_analytics_workspace_data_source.go

```diff
@@ -93,13 +95,12 @@ func dataSourceLogAnalyticsWorkspaceRead(d *schema.ResourceData, meta interface{
 	}
 
 	d.Set("workspace_id", resp.CustomerID)
-	d.Set("portal_url", resp.PortalURL)
```

Comment:
we still need to set this to an empty value for now, or it'll break existing users if this is `nil`